core/lib/utils/html/html_interaction.dart

```diff
@@ -405,4 +405,18 @@ class HtmlInteraction {
       }
     </script>
   ''';
+
+  static String scriptsHandleIframeClickListener(String viewId) => '''
+    <script type="text/javascript">
+      document.addEventListener('click', function (e) {
+        try {
+          const payload = {
+            view: '$viewId',
```

Comment:
how about to use `escape viewId` ?

Reply:
We don't need to escape viewId because viewId is randomly generated every time we open a new composer